src/Core/BuildTools/GLXmlConvert/Construction/ParsingHelpers.cs

```diff
@@ -263,22 +259,44 @@ public static Count ParseCountSignature
                 return new Count(valueReferenceName);
             }
 
-            // Some counts are a value reference along with some sort of numerical operation - let's skip forward in
-            // the string until we hit the first invalid character, and see if the rest are just numbers or math ops
-            var dataBeforeFirstInvalidCharacter = new string(countData.TakeWhile(char.IsLetterOrDigit).ToArray());
```

Comment:
We should reinstate these 2 lines as I liked the if statements we had before. Split the parsing logic into its own function, and then do `if (SyntaxFacts.IsValidIdentifier(dataBeforeFirstInvalidCharacter) && IsMath(dataAfterFirstInvalidCharater)`

Reply:
Added a small commit to do this, does that work for you?

Reply:
Yes that works nicely :) open another PR as I've already merged this one :P